### DISCUSSION TITLE
"Paste Components" library linked in one Figma file is bugging out

### DISCUSSION THREAD

**Discussion by ammehta27** [twilio-labs/paste#2435] [General] [answered]
I have been using Paste Components library in a file and yesterday for some reason in the **Input field with suffix icon**, the icon got replaced across the file from a **search** icon to a **cursor**. After that when I tried to replace the icon back to Search, it was resized to a 13px icon. This incident is not occurring in any other files and when I copy and paste the same component to and from this affected file, they are normal everywhere else. Only for this one, something has broken in the library and how it is linking the icons to the input component. However, I cannot investigate what the problem is myself - there is nothing visible wrong or different in my file.

[Figma file](https://www.figma.com/file/khLCGT6tniogrc6dLchxD7/Emergency-Address-UX?node-id=2%3A2)

See screenshots below for more context
![Screen Shot 2022-05-24 at 2 03 21 PM](https://user-images.githubusercontent.com/104515803/170383105-5ce5fab8-c491-451f-baa2-10e7cc2e3b61.png)
![Screen Shot 2022-05-19 at 9 12 33 AM](https://user-images.githubusercontent.com/104515803/170383162-44bc20ef-83c8-449a-815f-1e5e2be7444c.png)

**Comment by serifluous** [comment, marked as answer]
(@loreina and I chatted with @ammehta27, and we've escalated the bug to Figma support)

Marking this discussion as answered for now and will follow up either here or in Slack based on Figma's reply.

**Reply by serifluous** [reply]
Response from Figma:

> Looks like this is related to a longstanding issue that the team is aware of that happens when an instance is wrapped in an auto layout frame.
> 
> To resolve this, you can set the icon inside your auto layout frame to "fill" instead of "fixed" width/height.

Added a ticket to our backlog to fix it on our side: https://issues.corp.twilio.com/browse/DSYS-3397
